docs/releases/2.1.txt

```diff
@@ -154,6 +154,8 @@ Models
 ~~~~~~
 
 * Models can now use ``__init_subclass__()`` from :pep:`487`.
+* The new :meth:`.QuerySet.explain` method allows displaying the execution
```

Comment:
Please add a blank line between the `*`.

Reply:
Done 👍